vanilla-framework/vanilla-framework
# Add updated slack icon

## Done
- Updated Slack icon under 'Getting help' section

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/docs
- Scroll to 'Getting help' and check that the icon has been updated

## Details
Fixes: #2153
